ProjectLighthouse/Types/Serialization/GameManualXml.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Xml.Serialization;
+
+namespace LBPUnion.ProjectLighthouse.Types.Serialization;
+
+public class GameManualXml : ILbpSerializable
```

Comment:
Now that I think about this more and look at other serializable types, the Game prefix is usually only used when there is a corresponding entity. Could try something like `LbpCustomXml` or `LbpSerializableText`